tox.ini

```diff
@@ -16,7 +16,7 @@ envlist =
 [testenv]
 extras = all
 deps = {[base]testdeps}
-commands = nosetests -sv tests
+commands = nosetests -sv tests {posargs}
```

Comment:
Where does `posargs` come from? Is this change only for manual runs on a local machine?

Reply:
It is only for local runs. For example, if I want to debug with pdb, I have to pass the `-s` flag to nosetest, so thanks to posargs I can use the command `tox -e py36 -- -s`.

Reply:
ok, good to know